core/src/test/java/org/elasticsearch/search/sort/FieldSortBuilderTests.java

```diff
@@ -0,0 +1,43 @@
+package org.elasticsearch.search.sort;
+
+import static org.hamcrest.Matchers.stringContainsInOrder;
+
+import java.util.Arrays;
+
+import org.elasticsearch.common.xcontent.XContentBuilder;
+import org.elasticsearch.common.xcontent.XContentFactory;
+import org.elasticsearch.common.xcontent.XContentType;
+import org.elasticsearch.test.ESTestCase;
+
+public class FieldSortBuilderTests extends ESTestCase {
+
+    private static final String FIELDNAME = "myfield";
+    private static final String QUOTES = "\"";
+
+    public void testToXContent() throws Exception {
```

Comment:
I don't think you need to explicitely test toXContent, this is done by toString implicitely and this is the change that we want to check.

Reply:
Ok. As FieldSortBuilder test didn't exist, I used it as an already-to-develop test before to start fixing the failing one. I didn't even think in remove it, but indeed it's not a needed test for this change.